pkgs/development/python-modules/docstring-to-markdown/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-cov
+, pytest-flake8
+, mypy
+}:
+
+buildPythonPackage rec {
+  pname = "docstring-to-markdown";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "python-lsp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XVTlSqqWmvnB5nvjvgGDJmg71KKTq2hHB4//QW7ugvA=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-cov
+    pytest-flake8
+    mypy
```

Comment:
```suggestion
```

There is no advantage of running those. Please patch out the requirement.

Reply:
> There is no advantage of running those. Please patch out the requirement.

You mean to say there's no value at all in running `pytest`? These pytest deps are needed for pytest to run. `mypy` is not needed and I removed it.

Reply:
@SuperSandro2000 I think I understood now what you meant. I removed the `pytest-cov` and `pytest-flake8` deps by adding a patch.